Gekko/UI/MainViewController.swift

```diff
@@ -198,27 +228,43 @@ class MainViewController : UIViewController,
         }
     }
 
-    fileprivate func scheduleOrdersUpdating(onCompletion: @escaping () -> Void) {
-        handleOrdersUpdating(forPair:.BtcUah)
-        handleOrdersUpdating(forPair:.EthUah)
-        handleOrdersUpdating(forPair:.LtcUah)
-
+    fileprivate func scheduleOrdersUpdating() {
+        handleOrdersUpdating(onCompletion: {})
+        
         DispatchQueue.main.asyncAfter(deadline:DispatchTime.now() + MainViewController.PricePollTimeout) {
             [weak self] () in
             if (self != nil) {
-                self!.scheduleOrdersUpdating{}
+                self!.scheduleOrdersUpdating()
             }
         }
     }
 
-    fileprivate func handleOrdersUpdating(forPair pair:BTCTradeUACurrencyPair) {
+    fileprivate func handleOrdersUpdating(onCompletion:@escaping () -> Void) {
+        let RequiredOperationsCount = 6
```

Comment:
Not obvious why '6'. Please set to '3' and calculate responses inside 'handleOrdersUpdating'